example_config.json

```diff
@@ -1,6 +1,7 @@
 {
 	"TemplateData": {
 		"HBIR": {
+            "jar": "UPDATE ME",
```

Comment:
Does this need to be updated?

Reply:
Ah, it's updated when the user runs the GUI and selects their desired SPECjbb jar. It updates the path to the jar in the file (when user saves to file), but most importantly it saves it in memory, which is used for invoking